api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImpl.java

```diff
@@ -26,21 +27,21 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+@Getter(AccessLevel.PROTECTED)
 @Component
 @Transactional
 @Setter(AccessLevel.PACKAGE)
-public class FhirPersonServiceImpl implements FhirPersonService {
+public class FhirPersonServiceImpl extends BaseFhirService<Person, org.openmrs.Person> implements FhirPersonService {
```

Comment:
The `fhirPersonDao` should be renamed `dao`. The `personTranslator` should be renamed `translator`.